python/tvm/relay/frontend/keras.py

```diff
@@ -123,10 +124,11 @@ def _convert_advanced_activation(inexpr, keras_layer, etab):
         if isinstance(axis, list):
             raise tvm.error.OpAttributeUnImplemented(
                 'Softmax with axes {} is not supported.'.format(axis))
-        if axis == -1:
-            axis = 1
-        else:
-            axis = axis + 1 if axis < dims - 1 else 1
+        if etab.data_layout == 'NCHW':
```

Comment:
Maybe silly question but need to ask: are there no expected adjusts to be made on `axis` here when layout is not `NCHW`?

Reply:
TVM Keras frontend supports tensorflow backend only and the data_format attribute is channel_last (aka NHWC or NDHWC). So when the target layout is set NCHW (as default), the axis value should be transformed to be consist with the target layout.

Reply:
Understood - thanks